sound/soc/aml/m8/aml_i2s.c

```diff
@@ -90,14 +90,14 @@ static const struct snd_pcm_hardware aml_i2s_hardware = {
 	    SNDRV_PCM_FMTBIT_S16_LE | SNDRV_PCM_FMTBIT_S24_LE |
 	    SNDRV_PCM_FMTBIT_S32_LE,
 
-	.period_bytes_min = 64,
-	.period_bytes_max = 32 * 1024 * 2,
-	.periods_min = 2,
-	.periods_max = 1024,
-	.buffer_bytes_max = 128 * 1024 * 2 * 2,
+	.period_bytes_min = 256,
+	.period_bytes_max = 32 * 1024,
+	.periods_min = 4,
+	.periods_max = 64,
+	.buffer_bytes_max = 32 * 1024 * 64,
```

Comment:
Please consider reverting this line as it introduces issues S905X using T9015 codec.